components/03-organisms/component-wrapper/yds-component-wrapper.twig

```diff
@@ -1,6 +1,19 @@
-<div data-component-width={{ component_width|default('max') }}>
+{% set component__base_class = 'component-wrapper' %}
+
+<div {{ bem(component__base_class, component__modifiers, component__blockname, component__extra_classes) }} data-component-width={{ component_width|default('max') }}>
```

Comment:
Whenever you need to tackle more attributes besides classes I'd suggest using `add_attributes` instead. This will make the component much more extendible by other components that use it. Something like ...
```
{% set component_wrapper__attributes = component_wrapper__attributes|default([]) %}

{% set component_wrapper__attributes = component_wrapper__attributes|merge({
  'class': bem(component__base_class, component__modifiers, component__blockname, component__extra_classes),
  'data-component-width': component_width|default('max'),
}) %}

<div {{ add_attributes(component_wrapper__attributes) }}>
```

Reply:
@callinmullaney Ah, yeah, good call! Thanks!